content/news/YYYY-MM-DD-mixxx-2-3-1-released.md

```diff
@@ -0,0 +1,14 @@
+title: Mixxx 2.3.1 released
+authors: Owen Williams
+tags: 2.3, 2.3.1, release announcement
+comments: yes
+date: 2021-07-19 14:55:00
```

Comment:
```suggestion
status: draft
```

CI will fill in the date after merge and set it to published.

Reply:
Also, please rename the file because the `YYYY-MM-DD` name is not matched by the script: https://github.com/mixxxdj/website/blob/ab58bfd229eb7403a3013c7b3135917497f5e5e6/tools/update_news_date.py#L98

Use `XXXX-XX-XX` instead.